### PR TITLE
B2b b2b returns

### PR DESCRIPTION
**Pull request by m-alama**
## PR Description

Create B2B tests for returns

### CODE REVIEW

File: tests/api/b2b/glue/returns_endpoints/negative.robot [hunk 1]
```diff
@@ -0,0 +1,51 @@
+*** Settings ***
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/returns_endpoints/negative.robot:1]: please fix folder structure. it should be tests/api/b2b/glue/return_endpoints/returns - and here should be your positive and negative files

File: tests/api/b2b/glue/returns_endpoints/positive.robot [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    ...  AND    Zed: login on Zed with provided credentials:    ${yves_user_email}    
+    ...  AND    Zed: go to second navigation item level:    Sales    Orders
+    ...  AND    Zed: click Action Button in a table for row that contains:    ${yves_user_email}    View 
+    When I send a POST request:     /returns     {"data":{"type":"returns","attributes":{"store":"${store_de}","returnItems":[{"salesOrderItemUuid":"${returnableSalesOrderItemUuid}","reason":"${return_reason_damaged}"}]}}}
+    Then Response status code should be:     201
+    And Response reason should be:     Created
+    And Response header parameter should be:    Content-Type    ${default_header_content_type}
+    And Response body parameter should be:    [data][type]    returns
+    And Response body parameter should be:    [data][attributes][store]    ${store_de}
+    And Response body parameter should be:    [data][attributes][customerReference]    ${yves_user_reference}
```
[the remark this answers is not in the document]
Comment (m-alama) [reply tests/api/b2b/glue/returns_endpoints/positive.robot]: Done

File: tests/api/b2b/glue/returns_endpoints/positive.robot [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+    ...  AND    I send a POST request:     /returns     {"data":{"type":"returns","attributes":{"store":"${store_de}","returnItems":[{"salesOrderItemUuid":"${returnableSalesOrderItemUuid}","reason":"${return_reason_damaged}"}]}}}
+    ...  AND    Save value to a variable:    [data][id]    returnId
+    When I send a GET request:    /returns/${returnId}?include=return-items
+    Then Response status code should be:     200
+    And Response reason should be:     Ok
+    And Response body parameter should be:    [data][type]    returns
+    And Response body parameter should be:    [data][id]    ${returnId}
+    And Response body parameter should be:    [data][attributes][returnReference]    ${returnId}
+    And Response body parameter should be:    [data][attributes][store]    ${store_de}
+    And Response body parameter should be:    [data][attributes][customerReference]    ${yves_user_reference}
```
[the remark this answers is not in the document]
Comment (m-alama) [reply tests/api/b2b/glue/returns_endpoints/positive.robot]: Done

File: tests/api/b2b/glue/returns_endpoints/positive.robot [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    ...  AND    Response status code should be:    201
+    ...  AND    Save value to a variable:    [included][0][items][0][uuid]    returnableSalesOrderItemUuid
+    ...  AND    Zed: login on Zed with provided credentials:    ${yves_user_email}    
+    ...  AND    Zed: go to second navigation item level:    Sales    Orders
+    ...  AND    Zed: click Action Button in a table for row that contains:    ${yves_user_email}    View
```
[the remark this answers is not in the document]
Comment (OKrivtsova) [reply tests/api/b2b/glue/returns_endpoints/positive.robot]: I'm haiving an issue with these UI tests, please fix them
[ ERROR ] Error in file '/Users/krivtsova/Documents/AutomationSpryker/robotframework-suite-tests/tests/api/b2b/glue/returns_endpoints/returns/positive.robot' on line 5: Resource file '../../../../../resources/common/common_zed.robot' does not exist.

File: tests/api/b2b/glue/returns_endpoints/returns/negative.robot [hunk 5]
```diff
@@ -0,0 +1,51 @@
+*** Settings ***
+Suite Setup    SuiteSetup
+Test Setup    TestSetup
+Resource    ../../../../../resources/common/common_api.robot
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/returns_endpoints/returns/negative.robot:4]: there are issues after moving to a new folder. please run tests before making a new commit

[ ERROR ] Error in file '/Users/krivtsova/Documents/AutomationSpryker/robotframework-suite-tests/tests/api/b2b/glue/returns_endpoints/returns/positive.robot' on line 5: Resource file '../../../../../resources/common/common_zed.robot' does not exist.
[ ERROR ] Error in file '/Users/krivtsova/Documents/AutomationSpryker/robotframework-suite-tests/tests/api/b2b/glue/returns_endpoints/returns/negative.robot' on line 4: Resource file '../../../../../resources/common/common_api.robot' does not exist.

there are also seem to be suite setup duplicates if you try using zed and api
Multiple keywords with name 'SuiteSetup' found. Give the full name of the keyword you want to use:
    common.SuiteSetup
    common_api.SuiteSetup

maybe we could just remove the UI part until it is properly done

Comment (m-alama) [reply tests/api/b2b/glue/returns_endpoints/returns/negative.robot]: Done

File: tests/api/b2b/glue/returns_endpoints/returns/negative.robot [hunk 6]
```diff
@@ -0,0 +1,51 @@
+*** Settings ***
+Suite Setup    SuiteSetup
```
Comment (OKrivtsova) [inline tests/api/b2b/glue/returns_endpoints/returns/negative.robot:2]: please merge master to your branch and add enabler as requested in the chat

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**